server/store/store_test.go

```diff
@@ -1167,3 +1170,118 @@ func testListConnectedUsers(t *testing.T, store *SQLStore, _ *plugintest.API) {
 	delErr = store.DeleteUserInfo(testutils.GetID() + "2")
 	assert.Nil(delErr)
 }
+
+func testGetCountOfConnectedUsers(t *testing.T, store *SQLStore, _ *plugintest.API) {
+	assert := assert.New(t)
+	store.encryptionKey = func() []byte {
+		return make([]byte, 16)
+	}
+
+	count, getErr := store.GetCountOfConnectedUsers()
+	assert.Equal(0, count)
+	assert.Nil(getErr)
+
+	token := &oauth2.Token{
+		AccessToken:  "mockAccessToken-1",
+		RefreshToken: "mockRefreshToken-1",
+	}
+
+	storeErr := store.SetUserInfo(testutils.GetID()+"1", testutils.GetTeamsUserID()+"1", token)
+	assert.Nil(storeErr)
+
+	storeErr = store.SetUserInfo(testutils.GetID()+"2", testutils.GetTeamsUserID()+"2", token)
```

Comment:
we can keep the token nil here to check we only count users with a non-empty token